src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java

```diff
@@ -1,33 +1,27 @@
 package com.google.checkstyle.test.chapter7javadoc.rule712paragraphs;
 
-// 2 violations 5 lines below:
-//  '\<p\> tag should be placed immediately before the first word'
-//  '\<p\> tag should be preceded with an empty line.'
+// violation 3 lines below '<p> tag should be preceded with an empty line.'
 /**
  * Some Javadoc.
  * <p>
  * /^ WARN/ Some Javadoc.<p>
  */
```

Comment:
we loosing violation, it is not ok for google style.
But we will handle it at https://github.com/checkstyle/checkstyle/issues/15716
It is ok to have this small regression, as it is false negative.

Reply:
until https://github.com/checkstyle/checkstyle/issues/15685 is fixed, this Check is actually not functional, as it skips all proper HTMLed javadoc.
So one more temporal false-negative is ok, nobody will notice it.